src/v2/view-builder/views/SelectAuthenticatorEnrollView.js

```diff
@@ -7,8 +7,8 @@ const Body = BaseForm.extend({
   },
   subtitle: function() {
     const subtitle = this.options.settings.get('brandName') ?
-      loc('oie.select.authenticators.enroll.subtitle.custom', 'login', [this.options.settings.get('brandName')]):
-      loc('oie.select.authenticators.enroll.subtitle', 'login');
+      loc('oie.select.authenticators.enroll.subtitle.2.custom', 'login', [this.options.settings.get('brandName')]):
```

Comment:
why create a new key instead of reusing.

Reply:
@haishengwu-okta inconsistent translations, https://oktawiki.atlassian.net/wiki/spaces/eng/pages/741293778/i18n+dev+process+From+Oct+2019+onwards#If-translations-already-exist-for-the-key%3A

Reply:
Given OIE is not widely adapted, I feel it's fine to change string in place.
The problem mentioned in the wiki is concern that English and translation are not stay in sync until next translation coming back (~1 month). Don't think it's problem for us?
@nikhilvenkatraman-okta  @pradeepdewda-okta any thoughts?

Reply:
Yeah, we should be able to update values for `oie` strings. 

Also just a note, the key name `oie.select.authenticators.enroll.subtitle.2.custom` doesn't seem ideal. We should try and avoid using numbers in the key names and use some thing more meaningful for new keys as these keys could potentially be used by customers for customization.

Reply:
Ok I'll update that wiki with this new info